go/libraries/doltcore/sqle/logictest/regressions.sql

```diff
@@ -0,0 +1,158 @@
+CREATE TABLE `mysql_results` (
+  `test_file` VARCHAR(255) NOT NULL,
+  `line_num` BIGINT NOT NULL,
+  `duration` BIGINT NOT NULL,
+  `query_string` LONGTEXT NOT NULL,
+  `result` VARCHAR(255) NOT NULL,
+  `error_message` LONGTEXT,
+  `version` VARCHAR(255) NOT NULL,
+  PRIMARY KEY (`test_file`, `line_num`)
+);
+CREATE TABLE `mysql_mean_results` (
+  `version` VARCHAR(255) NOT NULL,
+  `test_file` VARCHAR(255) NOT NULL,
+  `line_num` BIGINT NOT NULL,
+  `mean_duration` DOUBLE NOT NULL,
+  `result` LONGTEXT NOT NULL,
+  PRIMARY KEY (`test_file`,`line_num`)
+);
+CREATE TABLE `nightly_dolt_results` (
+  `test_file` VARCHAR(255) NOT NULL,
+  `line_num` BIGINT NOT NULL,
+  `duration` BIGINT NOT NULL,
+  `query_string` LONGTEXT NOT NULL,
+  `result` LONGTEXT NOT NULL,
+  `error_message` LONGTEXT,
+  `version` VARCHAR(255) NOT NULL,
+  PRIMARY KEY (`test_file`,`line_num`)
+);
+CREATE TABLE `nightly_dolt_mean_results` (
+  `version` VARCHAR(255) NOT NULL,
+  `test_file` VARCHAR(255) NOT NULL,
+  `line_num` BIGINT NOT NULL,
+  `mean_duration` DOUBLE NOT NULL,
+  `result` LONGTEXT NOT NULL,
+  PRIMARY KEY (`test_file`,`line_num`)
+);
+CREATE TABLE `releases_dolt_results` (
+  `test_file` VARCHAR(255) NOT NULL,
+  `line_num` BIGINT NOT NULL,
+  `duration` BIGINT NOT NULL,
+  `query_string` LONGTEXT NOT NULL,
+  `result` LONGTEXT NOT NULL,
+  `error_message` LONGTEXT,
+  `version` VARCHAR(255) NOT NULL,
+  PRIMARY KEY (`test_file`,`line_num`)
+);
+CREATE TABLE `releases_dolt_mean_results` (
+  `version` VARCHAR(255) NOT NULL,
+  `test_file` VARCHAR(255) NOT NULL,
+  `line_num` BIGINT NOT NULL,
+  `mean_duration` DOUBLE NOT NULL,
+  `result` LONGTEXT NOT NULL,
+  PRIMARY KEY (`test_file`,`line_num`)
+);
+create view `mysql_releases_duration_change`
```

Comment:
I don't understand why comparing dolt performance to MySQL performance like this is useful. I mean, yes it is useful to know where we are performing worse than mysql (most places), but as an automated process it isn't what I care about. I care about dolt's performance getting worse over time (releases_nightly_duration_change), not how we do against mysql over time.